tools/find-files.ini

```diff
@@ -0,0 +1,32 @@
+[all]
+\[a-z]*
+.gitattributes
+.gitignore
+
+[ignore]
+components/cjson
+components/esptool_py
+components/lwip
+components/ssl/axtls
+components/ssl/mbedtls
+components/ssl/wolfssl
+tools/kconfig
+tools/kconfig_new
```

Comment:
```
.git
components/cjson
components/esp8266/include/xtensa
components/esptool_py
components/freertos/freertos
components/freertos/include
components/lwip/lwip
components/mqtt/paho
components/newlib/newlib/include
components/spiffs
components/ssl/axtls
components/ssl/mbedtls/mbedtls
components/ssl/wolfssl/wolfssl
tools/ci
tools/cmake
tools/kconfig
tools/kconfig_new
```

Reply:
I've updated the list except:

.git is not included by the all pattern.
components/ssl/mbedtls/mbedtls, tools/ci and tools/cmake doesn't use tabs.

Reply:
I tried, it really handled .git folder.
```
$ tools/reformat -e 
Replacing tabs with spaces...
Fixed .git/config
Fixed .git/hooks/commit-msg.sample
Fixed .git/hooks/pre-commit.sample
Fixed .git/hooks/pre-push.sample
Fixed .git/hooks/pre-rebase.sample
Fixed .git/hooks/pre-receive.sample
Fixed .git/hooks/prepare-commit-msg.sample
Fixed .git/hooks/update.sample
Fixed .git/index
Fixed .git/logs/HEAD
```
it will destroy git, i need to clone again.